src/views/search-page/stores/SearchLocalStore.ts

```diff
@@ -32,16 +32,17 @@ export const SearchLocalStore = (source: ISourceProps) => ({
   async _requestData() {
     const endpoint = this.endpoint.replace(':searchTerm', this.currentSearchTerm);
 
+    // requestAction(callback, api, ...tupleOfApiResponseTransforms)
     await requestAction(
       (status) => {
         this.searchResults = {
           ...status,
-          data: status.data ? status.data.map((model) => model.show) : [],
+          data: status.data ? status.data.results : [],
         };
       },
-      http.get<ShowsSearchResponseModel[]>(endpoint),
-      responseToModels<ShowsSearchResponseModel[]>(ShowsSearchResponseModel),
+      http.get<PeopleSearchResponse>(endpoint),
+      normalizeResponse<PeopleSearchResponse<People>>(),
       toastResponseError
-    );
+    ); // Trying to get the last return type would be "PeopleSearchResponse<People>"
```

Comment:
My goal is to have `status` on line 36 to be the last type (`PeopleSearchResponse<People>`) provided on line 43. Line 43 is the api call and line 44 and after that would you different function to transform the response data or handle the error object.

The code works how I want it but I am having a hard time trying to get the typing figured out.

This is an extract of the `requestAction` function:
```
requestAction = async <T>(callback: (status: UnknownResponseStatus<T>) => void, effect: Promise<APIResponse<T>>, ...transformers: APITransformer<T>[]): Promise<UnknownResponseStatus<T>> => {
```